dev/com.ibm.ws.jbatch.open_fat/fat/src/batch/fat/junit/CDITestCheckpoint.java

```diff
@@ -41,7 +41,6 @@ public class CDITestCheckpoint extends BatchFATHelper {
     @BeforeClass
     public static void setup() throws Exception {
 
-        server = LibertyServerFactory.getLibertyServer("checkpointbatchFAT");
         // Just happens to be a config that works we could reuse.  Could rename.
         BatchFATHelper.setConfig("CDITestCheckpoint/server.xml", testClass);
```

Comment:
Originally I had suggested that we continue to use the `checkpointbatchFAT` server but move the file `dev/com.ibm.ws.jbatch.open_fat/publish/files/CDITestCheckpoint/server.xml` to `dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/server.xml` and remove this `setConfig` call.

I think that would be more clean than copying the config over into the `batchFAT` server assigned by the super class.